wpilibj/src/test/java/edu/wpi/first/wpilibj/shuffleboard/ShuffleboardInstanceTest.java

```diff
@@ -16,20 +16,19 @@
 import static org.junit.jupiter.api.Assertions.assertAll;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
-@SuppressWarnings("MemberName")
 public class ShuffleboardInstanceTest {
-  private NetworkTableInstance ntInstance; // NOPMD
-  private ShuffleboardInstance shuffleboardInstance;
+  private NetworkTableInstance m_ntInstance; // NOPMD could be local variable
```

Comment:
Isn't PMD correct?

Reply:
Actually, I think you need to add an after each method and call close

Reply:
Ah, good catch